test/src/module/command/archivePushTest.c

```diff
@@ -851,7 +876,7 @@ testRun(void)
             "P00   INFO: push 1 WAL file(s) to archive: 000000010000000100000002\n"
             "P01   WARN: WAL file '000000010000000100000002' already exists in the repo1 archive with the same checksum\n"
             "            HINT: this is valid in some recovery scenarios but may also indicate a problem.\n"
-            "            WAL file '000000010000000100000002' already exists in the repo3 archive with the same checksum\n"
+            "P01   WARN: WAL file '000000010000000100000002' already exists in the repo3 archive with the same checksum\n"
             "            HINT: this is valid in some recovery scenarios but may also indicate a problem.\n"
             "P01 DETAIL: pushed WAL file '000000010000000100000002' to the archive");
```

Comment:
This isn't new, but I was curious as to why this is a DETAIL level
`"P01 DETAIL: pushed WAL file '000000010000000100000002' to the archive"`
but on line 575 it is an INFO level:
`"P00   INFO: pushed WAL file '000000010000000100000002' to the archive"`

Is it because the Async process already spits out 
`P00   INFO: push 1 WAL file(s) to archive: 000000010000000100000002`

Reply:
It is detail level in the async log to cut down on logging. It is info level in the foreground log because it's a lot of logging anyway, and info level is usually disabled for the foreground.